src/dmclock/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ if(K_WAY_HEAP)
 endif()
 
 if (NOT(TARGET gtest AND TARGET gtest_main))
-  find_package(gtest REQUIRED)
+  find_package(GTest REQUIRED)
```

Comment:
we do include GTest as a submodule. can we point `GTEST_ROOT` to the path in the source tree, so `FindGTest.cmake` can find it?